src/renderers/dom/client/ReactBrowserEventEmitter.js

```diff
@@ -212,8 +212,8 @@ var ReactBrowserEventEmitter = assign({}, ReactEventEmitterMixin, {
   listenTo: function(registrationName, contentDocumentHandle) {
     var mountAt = contentDocumentHandle;
     var isListening = getListeningForDocument(mountAt);
-    var dependencies = EventPluginRegistry.
-      registrationNameDependencies[registrationName];
+    var dependencies = EventPluginRegistry
+      .registrationNameDependencies[registrationName];
```

Comment:
We'll prefer a newline before splitting like this (except in cases where we're chaining) if it fits in 80. Let's just do that.

```js
    var dependencies = 
      EventPluginRegistry.registrationNameDependencies[registrationName];
```